### PR TITLE
corehunt: init at 4.2.0

### PR DESCRIPTION
**Pull request by dan4ik605743**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Relase notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by davidak** [comment]
Result of `nixpkgs-review pr 131093` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>corehunt</li>
  </ul>
</details>

File: pkgs/applications/misc/corehunt/default.nix [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+  buildInputs = [
+    qtbase
+    libcprime
+  ];
+
+  meta = with lib; {
+    description = "A file finder utility from the C Suite";
+    homepage = "https://gitlab.com/cubocore/coreapps/corehunt";
+    license = licenses.gpl3Plus;
```
Comment (davidak) [inline pkgs/applications/misc/corehunt/default.nix:27]: where do you got the `Plus`?

https://gitlab.com/cubocore/coreapps/corehunt/-/blob/master/LICENSE

**Review by davidak — CHANGES_REQUESTED** [review]
Looks good

Builds, runs

Can get merged when the license is clarified.

**Comment by dan4ik605743** [comment]
shit

**Comment by davidak** [comment]
what happened?

**Comment by dan4ik605743** [comment]
> what happened?

I'm a stranger who brought squash here

**Pull request closed without merging**